.vscode/settings.json

```diff
@@ -40,5 +40,15 @@
   ],
   "go.alternateTools": {
     "customFormatter": "${workspaceFolder}/core/fmt.sh",
-  }
+  },
+  "wake.compiler.solc.remappings": [
```

Comment:
Should this be in this PR?

Reply:
It doesn't hurt. I added it in case someone tries out the Ackee Solidity extension and doesn't know a specific command needs to be run to import the remappings.